[Regression] Template issues testing with new rnv project(rnv:0.37.0-canary.9, template: 0.21.1-alpha.8)

Describe the bug

Rnv does not install apps.

To Reproduce:

1. Create new project with rnv new and choose @flexn/template
2. Do the following commands (check all ticket)

Expected behavior:
All apps launches.

Screenshots or copy&paste:
1. `rnv run`, `rnv run -p tvos`, `rnv run -p androidtv`
```
 🚀  SUMMARY | 5/29/2023, 10:16:14 AM                                        │
│  $ rnv run                                                                   │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.37.0-canary.9                                           │
│  Project Name ($package.name): flexn_temp_a8                                 │
│  Project Version ($package.version): 0.1.0                                   │
│  Workspace ($.workspaceID): rnv                                              │
│  App Config (-c): app                                                        │
│  Build Scheme (-s): debug                                                    │
│  Env Info: darwin | x64 | node v16.14.2                                      │
│  Executed Time: 0h:0m:0s:492ms                                               │
│                                                                              │
│ [ error ] Cannot find module 'metro-cache'
Require stack:
- /Users/gabriele/Desktop/flexn_temp_a8/node_modules/@rnv/engine-rn-tvos/dist/adapter.js
- /Users/gabriele/Desktop/flexn_temp_a8/node_modules/@rnv/engine-rn-tvos/dist/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/core/engineManager/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/core/runtimeManager/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/cli/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/bin/index.js
Error: Cannot find module 'metro-cache'
Require stack:
- /Users/gabriele/Desktop/flexn_temp_a8/node_modules/@rnv/engine-rn-tvos/dist/adapter.js
- /Users/gabriele/Desktop/flexn_temp_a8/node_modules/@rnv/engine-rn-tvos/dist/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/core/engineManager/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/core/runtimeManager/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/cli/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/dist/index.js
- /Users/gabriele/.config/yarn/global/node_modules/rnv/bin/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gabriele/Desktop/flexn_temp_a8/node_modules/@rnv/engine-rn-tvos/src/adapter.js:5:19)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘
```
2. `rnv run -p web`
```
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  SUMMARY | 5/29/2023, 10:36:18 AM                                        │
│  $ rnv run -p web                                                            │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.37.0-canary.9                                           │
│  Project Name ($package.name): crt                                           │
│  Project Version ($package.version): 0.1.0                                   │
│  Workspace ($.workspaceID): flexn                                            │
│  Platform (-p): web                                                          │
│  Engine ($.platforms.web.engine): engine-rn-next                             │
│  Template: @flexn/template@0.21.1-alpha.8                                    │
│  App Config (-c): app                                                        │
│  Build Scheme (-s): debug                                                    │
│  Supported Platforms: web                                                    │
│  Env Info: darwin | x64 | node v16.14.2                                      │
│  Executed Time: 0h:0m:7s:866ms                                               │
│                                                                              │
│ [ warn ] [project configure] Plugin scope @flexn/plugins does not exists in package.json.
│ [ warn ] [project configure] This project uses encrypted files but you don't have them installed
│ [ warn ] [app configure] Plugin scope @flexn/plugins does not exists in package.json.
│ [ warn ] [project configure] Plugin react is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-art is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-dom is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-web is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin next is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @flexn/create is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @rnv/renative is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-tvos is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-windows is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-vector-icons is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @lightningjs/sdk is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @lightningjs/cli is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-community/cli-platform-ios is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-community/cli is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-windows/cli is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-navigation is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-navigation/elements is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-screens is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-safe-area-context is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-reanimated is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-masked-view/masked-view is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-media-query is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @flexn/shopify-flash-list is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @flexn/recyclerlistview is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-gesture-handler is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-art is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-dom is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-web is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin next is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @flexn/create is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @rnv/renative is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-tvos is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-windows is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-vector-icons is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @lightningjs/sdk is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @lightningjs/cli is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @react-native-community/cli-platform-ios is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @react-native-community/cli is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @react-native-windows/cli is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @react-navigation is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @react-navigation/elements is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-screens is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-safe-area-context is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-reanimated is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @react-native-masked-view/masked-view is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-media-query is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @flexn/shopify-flash-list is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin @flexn/recyclerlistview is not recognized plugin in flexn scope
│ [ warn ] [run] Plugin react-native-gesture-handler is not recognized plugin in flexn scope
│ [ error ] [run] COMMAND:

npx next dev --port 8080

FAILED with ERROR:

Command failed with exit code 1: npx next dev --port 8080

Error: Command failed with exit code 1: npx next dev --port 8080
    at makeError (/Users/pauliusguzas/Desktop/crt/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/pauliusguzas/Desktop/crt/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)


│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/app_web                                                    │
└──────────────────────────────────────────────────────────────────────────────┘
pauliusguzas@Pauliuss-MBP crt %
```
3. `rnv run -p android`
```
Dev server running at: http://0.0.0.0:8083/index.bundle?platform=android


[ hook ] [start] executePipe ('start:after')
[ task ] <= [run] <= start
⠋ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[1]
[ task ] [run] _isBundlerRunning[2] (NO)
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/pauliusguzas/Desktop/crt/null/local-cli/cli.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
COMMAND:

node null/local-cli/cli.js start --port 8083 --config=metro.config.js --no-interactive

FAILED with ERROR:

Command failed with exit code 1: node null/local-cli/cli.js start --port 8083 --config=metro.config.js --no-interactive

Error: Command failed with exit code 1: node null/local-cli/cli.js start --port 8083 --config=metro.config.js --no-interactive
    at makeError (/Users/pauliusguzas/Desktop/crt/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/pauliusguzas/Desktop/crt/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)


⠹ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[3]
[ task ] [run] _isBundlerRunning[4] (NO)
⠼ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[5]
[ task ] [run] _isBundlerRunning[6] (NO)
⠧ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[7]
[ task ] [run] _isBundlerRunning[8] (NO)
⠏ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[9]
[ task ] [run] _isBundlerRunning[10] (NO)
⠹ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[11]
[ task ] [run] _isBundlerRunning[12] (NO)
⠼ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[13]
[ task ] [run] _isBundlerRunning[14] (NO)
⠧ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[15]
[ task ] [run] _isBundlerRunning[16] (NO)
⠏ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[17]
[ task ] [run] _isBundlerRunning[18] (NO)
⠙ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[19]
[ task ] [run] _isBundlerRunning[20] (NO)
⠼ Waiting for bundler to finish...[ task ] [run] _isBundlerRunning[21]
[ task ] [run] _isBundlerRunning[22] (NO)
✖ Can't connect to bundler. Try restarting it.
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  SUMMARY | 5/29/2023, 10:46:29 AM                                        │
│  $ rnv run -p android                                                        │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.37.0-canary.9                                           │
│  Project Name ($package.name): crt                                           │
│  Project Version ($package.version): 0.1.0                                   │
│  Workspace ($.workspaceID): flexn                                            │
│  Platform (-p): android                                                      │
│  Engine ($.platforms.android.engine): engine-rn                              │
│  Template: @flexn/template@0.21.1-alpha.8                                    │
│  App Config (-c): app                                                        │
│  Build Scheme (-s): debug                                                    │
│  Target (-t): Nexus_5X_API_26                                                │
│  Supported Platforms: android                                                │
│  Env Info: darwin | x64 | node v16.14.2                                      │
│  Executed Time: 0h:0m:10s:994ms                                              │
│                                                                              │
│ [ warn ] [project configure] Plugin scope @flexn/plugins does not exists in package.json.
│ [ warn ] [project configure] This project uses encrypted files but you don't have them installed
│ [ warn ] [app configure] Plugin scope @flexn/plugins does not exists in package.json.
│ [ warn ] [project configure] Plugin react is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-art is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-dom is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-community/cli-platform-ios is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-community/cli is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @flexn/create is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @rnv/renative is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-web is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-tvos is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-windows is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-vector-icons is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @lightningjs/sdk is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @lightningjs/cli is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin next is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-windows/cli is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-navigation is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-navigation/elements is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-screens is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-safe-area-context is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-reanimated is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @react-native-masked-view/masked-view is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-media-query is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @flexn/shopify-flash-list is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin @flexn/recyclerlistview is not recognized plugin in flexn scope
│ [ warn ] [project configure] Plugin react-native-gesture-handler is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-art is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-dom is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @react-native-community/cli-platform-ios is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @react-native-community/cli is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @flexn/create is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @rnv/renative is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-web is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-tvos is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-windows is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-vector-icons is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @lightningjs/sdk is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @lightningjs/cli is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin next is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @react-native-windows/cli is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @react-navigation is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @react-navigation/elements is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-screens is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-safe-area-context is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-reanimated is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @react-native-masked-view/masked-view is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-media-query is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @flexn/shopify-flash-list is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin @flexn/recyclerlistview is not recognized plugin in flexn scope
│ [ warn ] [configure] Plugin react-native-gesture-handler is not recognized plugin in flexn scope
│ [ error ] [run] Can't connect to bundler. Try restarting it.
│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/app_android                                                │
└──────────────────────────────────────────────────────────────────────────────┘
pauliusguzas@Pauliuss-MBP crt %
```
4. `rnv run -p ios`, `rnv run -p macos`
````
│ [ error ] [configure] COMMAND:

pod update

FAILED with ERROR:

### Error
```
URI::BadURIError - relative URI: /Users/pauliusguzas/Desktop/crt/platformBuilds/app_ios/null/third-party-podspecs/DoubleConversion.podspec
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/uri/generic.rb:1490:in `find_proxy'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:195:in `block in open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:212:in `block in open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:210:in `catch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:210:in `open_loop'
[!] Oh no, an error occurred.
Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=relative+URI%3A+%2FUsers%2Fpauliusguzas%2FDesktop%2Fcrt%2FplatformBuilds%2Fapp_ios%2Fnull%2Fthird-party-podspecs%2FDoubleConversion.podspec&type=Issues
If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new
Be sure to first read the contributing guide for details on how to properly submit a ticket:

Command failed with exit code 1: pod update
Update all pods
Updating local specs repositories

CocoaPods 1.12.1 is available.
To update use: `sudo gem install cocoapods`

For more information, see https://blog.cocoapods.org and the CHANGELOG for this version at https://github.com/CocoaPods/CocoaPods/releases/tag/1.12.1

Analyzing dependencies
Fetching podspec for `DoubleConversion` from `null/third-party-podspecs/DoubleConversion.podspec`

――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/pod update
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?


### Stack

```
   CocoaPods : 1.11.3
        Ruby : ruby 3.1.1p18 (2022-02-18 revision 53f5fc4236) [x86_64-darwin20]
    RubyGems : 3.3.7
        Host : macOS 13.2 (22D49)
       Xcode : 14.2 (14C18)
         Git : git version 2.37.1 (Apple Git-137.1)
Ruby lib dir : /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib
Repositories : cocoapods - git - https://github.com/CocoaPods/Specs.git @ 3562657e243e5b5db71f27bcfb485159191d00a2

               trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0
```

### Podfile

```ruby
platform :ios, '12.0'
use_frameworks!


# ReNative: To allow dynamic plugins via flavours autolinking has to be disabled
# require_relative '../../node_modules/@react-native-community/cli-platform-ios/native_modules'

target 'RNVApp' do
  pod 'FBLazyVector', :path => "null/Libraries/FBLazyVector"
  pod 'FBReactNativeSpec', :path => "null/React/FBReactNativeSpec"
  pod 'RCTRequired', :path => "null/Libraries/RCTRequired"
  pod 'RCTTypeSafety', :path => "null/Libraries/TypeSafety"
  pod 'React', :path => 'null/'
  pod 'React-Core', :path => 'null/'
  pod 'React-CoreModules', :path => 'null/React/CoreModules'
  pod 'React-Core/DevSupport', :path => 'null/'
  pod 'React-RCTActionSheet', :path => 'null/Libraries/ActionSheetIOS'
  pod 'React-RCTAnimation', :path => 'null/Libraries/NativeAnimation'
  pod 'React-RCTBlob', :path => 'null/Libraries/Blob'
  pod 'React-RCTImage', :path => 'null/Libraries/Image'
  pod 'React-RCTLinking', :path => 'null/Libraries/LinkingIOS'
  pod 'React-RCTNetwork', :path => 'null/Libraries/Network'
  pod 'React-RCTSettings', :path => 'null/Libraries/Settings'
  pod 'React-RCTText', :path => 'null/Libraries/Text'
  pod 'React-RCTVibration', :path => 'null/Libraries/Vibration'
  pod 'React-Core/RCTWebSocket', :path => 'null/'

  pod 'React-cxxreact', :path => 'null/ReactCommon/cxxreact'
  pod 'React-jsi', :path => 'null/ReactCommon/jsi'
  pod 'React-jsiexecutor', :path => 'null/ReactCommon/jsiexecutor'
  pod 'React-jsinspector', :path => 'null/ReactCommon/jsinspector'
  pod 'React-callinvoker', :path => "null/ReactCommon/callinvoker"
  pod 'ReactCommon/turbomodule/core', :path => "null/ReactCommon"
  pod 'Yoga', :path => 'null/ReactCommon/yoga'

  pod 'DoubleConversion', :podspec => 'null/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => 'null/third-party-podspecs/glog.podspec'
  pod 'boost', :podspec => 'null/third-party-podspecs/boost.podspec'
  pod 'RCT-Folly', :podspec => 'null/third-party-podspecs/RCT-Folly.podspec'

  pod 'React-perflogger', :path => 'null/ReactCommon/reactperflogger'
  pod 'React-logger', :path => 'null/ReactCommon/logger'
  pod 'React-runtimeexecutor', :path => 'null/ReactCommon/runtimeexecutor'



  # ReNative: To allow dynamic plugins via flavours autolinking has to be disabled
  # use_native_modules!('../..')
end

pre_install do |installer|
  # workaround for https://github.com/CocoaPods/CocoaPods/issues/3289
  Pod::Installer::Xcode::TargetValidator.send(:define_method, :verify_no_static_framework_transitive_dependencies) {}
end

static_frameworks = []

pre_install do |installer|
  installer.pod_targets.each do |pod|
    if static_frameworks.include?(pod.name)
      def pod.static_framework?;
        true
      end
      def pod.build_type;
        Pod::BuildType.static_library
      end
    end
  end
end


post_install do |installer|

end
```

### Error

```
URI::BadURIError - relative URI: /Users/pauliusguzas/Desktop/crt/platformBuilds/app_ios/null/third-party-podspecs/DoubleConversion.podspec
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/uri/generic.rb:1490:in `find_proxy'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:195:in `block in open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:212:in `block in open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:210:in `catch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:210:in `open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:151:in `open_uri'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/external_sources/podspec_source.rb:19:in `block in fetch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:86:in `titled_section'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/external_sources/podspec_source.rb:11:in `fetch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:993:in `fetch_external_source'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:972:in `block (2 levels) in fetch_external_sources'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:971:in `each'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:971:in `block in fetch_external_sources'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:64:in `section'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:970:in `fetch_external_sources'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:117:in `analyze'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:416:in `analyze'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:241:in `block in resolve_dependencies'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:64:in `section'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:240:in `resolve_dependencies'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:161:in `install!'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/command/update.rb:63:in `run'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/command.rb:52:in `run'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/bin/pod:55:in `<top (required)>'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/pod:25:in `load'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/pod:25:in `<main>'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `eval'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=relative+URI%3A+%2FUsers%2Fpauliusguzas%2FDesktop%2Fcrt%2FplatformBuilds%2Fapp_ios%2Fnull%2Fthird-party-podspecs%2FDoubleConversion.podspec&type=Issues

If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new

Be sure to first read the contributing guide for details on how to properly submit a ticket:
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md

Don't forget to anonymize any private data!

Looking for related issues on cocoapods/cocoapods...
Found no similar issues. To create a new issue, please visit:
https://github.com/cocoapods/cocoapods/issues/new

Error: Command failed with exit code 1: pod update
Update all pods
Updating local specs repositories

CocoaPods 1.12.1 is available.
To update use: `sudo gem install cocoapods`

For more information, see https://blog.cocoapods.org and the CHANGELOG for this version at https://github.com/CocoaPods/CocoaPods/releases/tag/1.12.1

Analyzing dependencies
Fetching podspec for `DoubleConversion` from `null/third-party-podspecs/DoubleConversion.podspec`

――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/pod update
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?


### Stack

```
   CocoaPods : 1.11.3
        Ruby : ruby 3.1.1p18 (2022-02-18 revision 53f5fc4236) [x86_64-darwin20]
    RubyGems : 3.3.7
        Host : macOS 13.2 (22D49)
       Xcode : 14.2 (14C18)
         Git : git version 2.37.1 (Apple Git-137.1)
Ruby lib dir : /Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib
Repositories : cocoapods - git - https://github.com/CocoaPods/Specs.git @ 3562657e243e5b5db71f27bcfb485159191d00a2

               trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0
```

### Podfile

```ruby
platform :ios, '12.0'
use_frameworks!


# ReNative: To allow dynamic plugins via flavours autolinking has to be disabled
# require_relative '../../node_modules/@react-native-community/cli-platform-ios/native_modules'

target 'RNVApp' do
  pod 'FBLazyVector', :path => "null/Libraries/FBLazyVector"
  pod 'FBReactNativeSpec', :path => "null/React/FBReactNativeSpec"
  pod 'RCTRequired', :path => "null/Libraries/RCTRequired"
  pod 'RCTTypeSafety', :path => "null/Libraries/TypeSafety"
  pod 'React', :path => 'null/'
  pod 'React-Core', :path => 'null/'
  pod 'React-CoreModules', :path => 'null/React/CoreModules'
  pod 'React-Core/DevSupport', :path => 'null/'
  pod 'React-RCTActionSheet', :path => 'null/Libraries/ActionSheetIOS'
  pod 'React-RCTAnimation', :path => 'null/Libraries/NativeAnimation'
  pod 'React-RCTBlob', :path => 'null/Libraries/Blob'
  pod 'React-RCTImage', :path => 'null/Libraries/Image'
  pod 'React-RCTLinking', :path => 'null/Libraries/LinkingIOS'
  pod 'React-RCTNetwork', :path => 'null/Libraries/Network'
  pod 'React-RCTSettings', :path => 'null/Libraries/Settings'
  pod 'React-RCTText', :path => 'null/Libraries/Text'
  pod 'React-RCTVibration', :path => 'null/Libraries/Vibration'
  pod 'React-Core/RCTWebSocket', :path => 'null/'

  pod 'React-cxxreact', :path => 'null/ReactCommon/cxxreact'
  pod 'React-jsi', :path => 'null/ReactCommon/jsi'
  pod 'React-jsiexecutor', :path => 'null/ReactCommon/jsiexecutor'
  pod 'React-jsinspector', :path => 'null/ReactCommon/jsinspector'
  pod 'React-callinvoker', :path => "null/ReactCommon/callinvoker"
  pod 'ReactCommon/turbomodule/core', :path => "null/ReactCommon"
  pod 'Yoga', :path => 'null/ReactCommon/yoga'

  pod 'DoubleConversion', :podspec => 'null/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => 'null/third-party-podspecs/glog.podspec'
  pod 'boost', :podspec => 'null/third-party-podspecs/boost.podspec'
  pod 'RCT-Folly', :podspec => 'null/third-party-podspecs/RCT-Folly.podspec'

  pod 'React-perflogger', :path => 'null/ReactCommon/reactperflogger'
  pod 'React-logger', :path => 'null/ReactCommon/logger'
  pod 'React-runtimeexecutor', :path => 'null/ReactCommon/runtimeexecutor'



  # ReNative: To allow dynamic plugins via flavours autolinking has to be disabled
  # use_native_modules!('../..')
end

pre_install do |installer|
  # workaround for https://github.com/CocoaPods/CocoaPods/issues/3289
  Pod::Installer::Xcode::TargetValidator.send(:define_method, :verify_no_static_framework_transitive_dependencies) {}
end

static_frameworks = []

pre_install do |installer|
  installer.pod_targets.each do |pod|
    if static_frameworks.include?(pod.name)
      def pod.static_framework?;
        true
      end
      def pod.build_type;
        Pod::BuildType.static_library
      end
    end
  end
end


post_install do |installer|

end
```

### Error

```
URI::BadURIError - relative URI: /Users/pauliusguzas/Desktop/crt/platformBuilds/app_ios/null/third-party-podspecs/DoubleConversion.podspec
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/uri/generic.rb:1490:in `find_proxy'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:195:in `block in open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:212:in `block in open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:210:in `catch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:210:in `open_loop'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/open-uri.rb:151:in `open_uri'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/external_sources/podspec_source.rb:19:in `block in fetch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:86:in `titled_section'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/external_sources/podspec_source.rb:11:in `fetch'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:993:in `fetch_external_source'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:972:in `block (2 levels) in fetch_external_sources'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:971:in `each'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:971:in `block in fetch_external_sources'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:64:in `section'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:970:in `fetch_external_sources'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/analyzer.rb:117:in `analyze'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:416:in `analyze'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:241:in `block in resolve_dependencies'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:64:in `section'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:240:in `resolve_dependencies'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:161:in `install!'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/command/update.rb:63:in `run'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/lib/cocoapods/command.rb:52:in `run'
/Users/pauliusguzas/.rvm/rubies/ruby-3.1.1/lib/ruby/gems/3.1.0/gems/cocoapods-1.11.3/bin/pod:55:in `<top (required)>'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/pod:25:in `load'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/pod:25:in `<main>'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `eval'
/Users/pauliusguzas/.rvm/gems/ruby-3.1.1/bin/ruby_executable_hooks:22:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=relative+URI%3A+%2FUsers%2Fpauliusguzas%2FDesktop%2Fcrt%2FplatformBuilds%2Fapp_ios%2Fnull%2Fthird-party-podspecs%2FDoubleConversion.podspec&type=Issues

If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new

Be sure to first read the contributing guide for details on how to properly submit a ticket:
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md

Don't forget to anonymize any private data!

Looking for related issues on cocoapods/cocoapods...
Found no similar issues. To create a new issue, please visit:
https://github.com/cocoapods/cocoapods/issues/new
    at makeError (/Users/pauliusguzas/Desktop/crt/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/pauliusguzas/Desktop/crt/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)


│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/app_ios/RNVApp.xcworkspace                                 │
└──────────────────────────────────────────────────────────────────────────────┘
pauliusguzas@Pauliuss-MBP crt %
````

--- a/packages/template/renative.template.json
+++ b/packages/template/renative.template.json
@@ -4,7 +4,8 @@
         "packageTemplate": {
             "devDependencies": {
                 "typescript": "4.9.5",
-                "@types/react": "18.2.6"
+                "@types/react": "18.2.6",
+                "@flexn/plugins": "1.0.8"
             }
         }
     },

